tests/e2e/util.go

```diff
@@ -6823,3 +6826,90 @@ func getVmdkPathFromVolumeHandle(sshClientConfig *ssh.ClientConfig, masterIp str
 	vmdkPath := result.Stdout
 	return vmdkPath
 }
+
+func checkVcServicesHealth(ctx context.Context, host string, timeout ...time.Duration) {
```

Comment:
In function definition what all parameters it is returning ?

Reply:
added